src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -176,8 +224,8 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         /// Appends the Message of an Exception to the specified <see cref="StringBuilder" />.
         /// </summary>
         /// <param name="sb">The <see cref="StringBuilder"/> to append the rendered data to.</param>
-        /// <param name="ex">The exception containing the Message to append.</param>
-        protected virtual void AppendMessage(StringBuilder sb, Exception ex)
+        /// <param name="ex">The exception containing the Message to append.</param>        
+        private static void AppendMessage(StringBuilder sb, Exception ex)
```

Comment:
For Adding this into the Dictionary we have to move this to Private. We cannot maintain both the instance and static methods also. 

The other solution which I could think of is making a internal static class and mapping them together.